src/spotify-web-api.js

```diff
@@ -1011,6 +1037,7 @@ function SpotifyWebApi(credentials) {
 
     _addBodyParameters(request, options);
```

Comment:
Missing check in `_addBodyParameters` if `options` contains a `headers` object, similar to the check in `_addQueryParameters`.